shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Rules/ClassifyTraces.hs

```diff
@@ -132,7 +136,7 @@ genesisLedgerState = Just $ mkGenesisLedgerState (geConstants (genEnv p))
     p :: Proxy (ShelleyEra c)
     p = Proxy
 
-relevantCasesAreCovered :: Property
+relevantCasesAreCovered :: (Arbitrary Coin) => Property
```

Comment:
These constraints are a bit wrong. `Coin` is a fixed type; there is either an instance or not. Haskell will in fact usually warn about such constraints here...